### DISCUSSION TITLE
Why does xl: text-3xl override sm: test-xs?

### DISCUSSION THREAD

**Discussion by jiapy97** [tailwindlabs/tailwindcss#5420] [Help] [answered]
![image](https://user-images.githubusercontent.com/57215916/132326731-478a67e5-2a35-4458-8fd1-b6c536b69d7b.png)

**Comment by adbir** [comment, marked as answer]
Why do you have a space between the breakpoint and the style?

**Reply by jiapy97** [reply]
I undestand,thank you ，I would also like to ask a question about what happens if the screen size is smaller than the set sm in the case of mobile?
